docker.Makefile

```diff
@@ -22,7 +22,7 @@ PYTHON_VERSION            = 3.7
 PYTORCH_VERSION           = $(shell git describe --tags --always)
 # Can be either official / dev
 BUILD_TYPE                = dev
-BUILD_PROGRESS            = auto
+BUILD_PROGRESS           ?= auto
```

Comment:
Changed so we can pass BUILD_PROGRESS as an env variable from the actions workflow